connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -138,7 +138,9 @@ private void pause() {
 
                 case STARTED:
                     connector.stop();
-                    // fall through
```

Comment:
findbugs has an LGPL license, which is a Class X license according to Apache.  We cannot add the suppressFBwarnings annotations unless we add a Gradle dependency on this LGPL library.  Of course, we could argue that we are not distributing the findbugs jars that we have a compile-time dependency on.  But it's an awkward argument to make, and there is always the risk that we could accidentally ship one of these jars if the project has the compile-time dependency.  It is simpler and better just to use the suppression file-- the same way we do with checkstyle.

If it becomes too much of a burden, we could completely disable the rule about switch statement fallthroughs if we wanted to.  It's pretty simple-- just have this clause:
```
    <Match>
        <Bug pattern="SF_SWITCH_FALLTHROUGH"/>
    </Match>
```

So far it has been pretty easy to suppress or fix any cases like this.  There are less than half a dozen in the whole AK codebase.  So I don't have a strong opinion.  I would prefer to get this patch in, and then discuss at that point